common/src/main/java/earth/terrarium/prometheus/common/commands/admin/WarpCommand.java

```diff
@@ -55,7 +55,10 @@ private static LiteralArgumentBuilder<CommandSourceStack> add() {
             .requires(source -> source.hasPermission(2))
             .then(Commands.argument("name", StringArgumentType.greedyString())
                 .executes(context -> {
-                    WarpHandler.add(context.getSource().getPlayerOrException(), StringArgumentType.getString(context, "name"));
+                    ServerPlayer player = context.getSource().getPlayerOrException();
+                    String warpName = StringArgumentType.getString(context, "name");
+                    WarpHandler.add(player, warpName);
```

Comment:
this need to return if add returns false 
```suggestion
                    if (!WarpHandler.add(player, warpName)) return 0;
```